routes/manual.tsx

```diff
@@ -324,7 +287,18 @@ export const handler: Handlers<Data> = {
         }),
     ]);
 
-    return render!({ tableOfContents, content, version });
+    const { pageList, redirectList } = generateToC(
+      tableOfContents,
+      `/manual@${version}`,
+    );
+
+    const slashPath = "/" + params.path;
```

Comment:
Seems it can't handle a situation when `params.path` starts with a slash.

Reply:
it never starts with a slash, so not a concern